### ISSUE TITLE
MPA router treating all links as external links

### ISSUE THREAD

**Issue by thescientist13** [ProjectEvergreen/greenwood#562]
## Type of Change
- [ ] New Feature Request
- [ ] Documentation / Website
- [ ] Improvement / Suggestion
- [x] Bug
- [ ] Other (please clarify below)

## Summary
MPA router, even when enabled now and "fixed" as part of #500 , is now not doing _any_ client side routing.  All links just do a full page reload.  :/

> _Just to preface that hopefully #559 will address this cleanly once and for all by _hopefully_ being able to target the `<a>` tag directly, combined with some investment in #560._

## Details
Looks like with the new [URL detection logic](https://github.com/ProjectEvergreen/greenwood/commit/bd49529b135b22de243456dff43dae3e8d8f8737#diff-e4b4bc56ca553ca4d6160f0710d5d2a6d75a1af1a3ee2235bff9cc8ea7931afaR10), while it was working locally on `localhost`, I would see something like _/docs/_,  In production, you'll see it actually returns as _https://www.greenwoodjs.io/docs_ which will of course pass the URL check.  Locally, if I edit my _etc/hosts_ and visit `local.greenwoodjs.io:8080` when running `yarn serve` I can reproduce the same issue as in production.

Basically, the best way I can figure out how to do this at the time, to distinguish between a "route" and an off site URL, is to see if the current link includes the current domain in it.  If it does, assume it is a route, otherwise consider it an external link.  🤷‍♂️ 

---

As you can tell this is not ideal, ideally for any given links, e.g.
```html
<a href="/docs/" ....>Greenwood Docs</a>

<a href="http://lit-element.polymer=project.org" ...>Lit Docs</a>
```

I would want to examine the _exact_ value of the href, not whatever the browser fully qualifies.
